packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useDraftOutboundLines.tsx

```diff
@@ -104,10 +108,9 @@ export const useDraftOutboundLines = (
     if (!data) return;
 
     setDraftOutboundLines(() => {
-      if (!lines) return [];
       const rows = data
         .map(batch => {
-          const invoiceLine = lines.find(
+          const invoiceLine = (lines ?? []).find(
```

Comment:
this and the above - part of a fix for no stocklines showing, when I know that there are some!